instant-distance-py/examples/translations/README.md

```diff
@@ -0,0 +1,45 @@
+# Translation Example
+
+This example utilizes the pre-trained aligned word vectors made available by
+Facebook Research as part of
+[fastText](https://fasttext.cc/docs/en/aligned-vectors.html) to translate
+English words to French and Italian.
+
+## Trying it out
+
+Python 3.9 is required.
```

Comment:
Currently we only publish macOS wheels. Should we mention that here? Or figure out what we need to publish Linux/Windows wheels?

Reply:
I didn't realize that. I'll update the messaging.